CHANGELOG.unreleased.md

```diff
@@ -10,5 +10,6 @@
 
 ### Released packages
 
+- xen-api v0.24.6
```

Comment:
@julien-f It should increment the patch or the minor value ?

Reply:
Patch.